pkg/scalers/datadog_scaler.go

```diff
@@ -30,45 +29,48 @@ type datadogScaler struct {
 	httpClient           *http.Client
 	logger               logr.Logger
 	useClusterAgentProxy bool
+	metricType           v2.MetricTargetType
 }
 
+// TODO: Need to check whether we can deprecate vType and how should we proceed with it
```

Comment:
@kedacore/keda-maintainers Hi team, I have one question and would need your help  :pray: . Are we ready to deprecate `vType` ? it has been flagged out in the  PR https://github.com/kedacore/keda/pull/3491 2 years ago https://github.com/kedacore/keda/blob/30ddff317784d8280a897f6befc8406b0d2ef5f5/pkg/scalers/datadog_scaler.go#L254-L265
I'm slightly confused as it is also metntioned as part of the test case in https://github.com/kedacore/keda/blob/30ddff317784d8280a897f6befc8406b0d2ef5f5/pkg/scalers/datadog_scaler_test.go#L111

Reply:
Thanks @wozniakjan ,for now I will just focus on the refactoring and leave this for another time